<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
    <size>
      <width>801</width>
      <height>618</height>
    </size>
  </property>
  <property name="windowTitle">
   <string>HedgeEdit</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="HedgeEdit::UI::ViewportWidget" name="viewport" native="true">
       <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
         </sizepolicy>
       </property>
       <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
       </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>30</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionImportXML"/>
    <addaction name="actionExportXML"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectNone"/>
    <addaction name="separator"/>
    <addaction name="actionSceneView"/>
    <addaction name="actionAssetsDialog"/>
    <addaction name="actionMaterialEditor"/>
    <addaction name="actionPropertiesPanel"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewSelected"/>
    <addaction name="actionDarkTheme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>270</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Properties Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="HedgeEdit::UI::PropertiesPanel" name="propertiesPanel" />
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save Sets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionImportXML">
   <property name="text">
    <string>&amp;Import XML...</string>
   </property>
  </action>
  <action name="actionExportXML">
   <property name="text">
    <string>&amp;Export XML...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select &amp;None</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSceneView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scene &amp;View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAssetsDialog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>As&amp;sets Dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMaterialEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Material Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionViewSelected">
   <property name="text">
    <string>View &amp;Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="text">
    <string>Debug Enable &amp;Dark Theme</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionPropertiesPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prop&amp;erties Panel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
  </connection>
  <connection>
   <sender>actionDarkTheme</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>DebugEnableDarkTheme()</slot>
  </connection>
 </connections>
</ui>
